server/database.py

```diff
@@ -9,8 +9,18 @@
 engine = create_engine(DATABASE_URL)
 db_session = scoped_session(sessionmaker(autocommit=False, autoflush=True, bind=engine))
 
+meta = MetaData(
+    naming_convention={
+        "ix": "%(column_0_N_label)s_idx",
+        "uq": "%(table_name)s_%(column_0_N_name)s_key",
+        "ck": "%(table_name)s_%(constraint_name)s_check",
+        "fk": "%(table_name)s_%(column_0_N_name)s_fkey",
+        "pk": "%(table_name)s_pkey",
+    }
+)
```

Comment:
ok so this is a tricky thing. this configures sqlalchemy to use these templates for naming constraints instead of using the postgresql auto-naming. this is necessary for alembic to be able to come up with constraint names that match the actual constraint names in the database when autogenerating migration code. i tried to match the postgresql constraint naming conventions as well as I could

more here: https://alembic.sqlalchemy.org/en/latest/naming.html

Reply:
totally sane reason for naming things a particular way :)